Dockerfile

```diff
@@ -2,8 +2,7 @@ FROM alpine:edge
 
 RUN apk update && apk add ca-certificates && rm -rf /var/cache/apk/*
 
-COPY out/pemtokeystore /pemtokeystore
+COPY --from=0 /go/bin/pemtokeystore /pemtokeystore
```

Comment:
This isn't a multistage Dockerfile?

Reply:
Its a PR for the changes I did in syndesis/pemtokeystore , thought you might be interested as it allows Dockerhub automated builds.

Reply:
Oops, my bad.

Reply:
oops, my bad.